cart/interfaces/controller/cartapicontroller.go

```diff
@@ -145,6 +145,30 @@ func (cc *CartAPIController) ApplyGiftCardAndGetAction(ctx context.Context, r *w
 	return cc.handlePromotionAction(ctx, r, "giftcard_error", cc.cartService.ApplyGiftCard)
 }
 
+// ApplyCombinedVoucherGift applies a given first to the voucher API Endpoint (if no voucher code is already set)
```

Comment:
since we don't know how the underlying cart adapter works we should adjust this comment.

Reply:
moved to cartservice interface